indexers/keyvalue/LevelDBIndexer/__init__.py

```diff
@@ -28,16 +28,17 @@ def get_create_handler(self):
         import plyvel
         return plyvel.DB(self.index_abspath, create_if_missing=True)
 
-    def add(self, objs):
+
+    def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
         """Add a JSON-friendly object to the indexer
 
         :param objs: objects can be serialized into JSON format
         """
         with self.write_handler.write_batch() as h:
-            for k, obj in objs.items():
-                key = k.encode('utf8')
-                value = json.dumps(obj).encode('utf8')
-                h.put(key, value)
+            for k, v in zip(keys, values):
+                # v = json.dumps(v).encode('utf8')
+                k = bytes(k)
+                h.put(k, v)
```

Comment:
I would do directly h.put(bytes(k), v).

I think os confusing to reassign values

Reply:
true, this should be avoided